opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/context/ServiceContextProviderConfig.java

```diff
@@ -0,0 +1,90 @@
+package de.adorsys.opba.protocol.facade.services.context;
+
+import de.adorsys.opba.db.repository.jpa.AuthorizationSessionRepository;
+import de.adorsys.opba.db.repository.jpa.BankProfileJpaRepository;
+import de.adorsys.opba.db.repository.jpa.ServiceSessionRepository;
+import de.adorsys.opba.protocol.facade.config.encryption.ConsentAuthorizationEncryptionServiceProvider;
+import de.adorsys.opba.protocol.facade.services.EncryptionKeySerde;
+import de.adorsys.opba.protocol.facade.services.fintech.FintechAuthenticator;
+import de.adorsys.opba.protocol.facade.services.scoped.RequestScopedProvider;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+public class ServiceContextProviderConfig {
+
+    @Bean(NoRedirectCodeValidationServiceContextProvider.NO_REDIRECT_CODE_VALIDATION)
+    NoRedirectCodeValidationServiceContextProvider serviceContextProviderNoRedirectCodeValidation(
```

Comment:
This bean should be activated only on certain profile

Reply:
This bean is used by DenyAuthorizationService with any profile